src/core/public/chrome/ui/header/collapsible_nav.test.tsx

```diff
@@ -76,19 +77,20 @@ function clickGroup(component: ReactWrapper, group: string) {
 describe('CollapsibleNav', () => {
   // this test is mostly an "EUI works as expected" sanity check
   it('renders the default nav', () => {
-    const onLock = sinon.spy();
-    const component = mount(<CollapsibleNav {...mockProps()} onIsLockedUpdate={onLock} />);
+    // Removing Locking ability without totally removing
```

Comment:
I'm not sure I fully understand this code comment. If it means that the default nav now doesn't lock to the page anymore, should we maybe change the test to be something along the lines of:
```
it('renders the default nav that is not locked', () => {
```